sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py

```diff
@@ -172,8 +172,7 @@ def __init__(self, user_id, **kwargs):
             self.raw_id = _microsoft_teams_user_raw_id(self)
 
 
-def _microsoft_teams_user_raw_id(identifier):
-    # type (MicrosoftTeamsUserIdentifier) -> str
+def _microsoft_teams_user_raw_id(identifier: MicrosoftTeamsUserIdentifier) -> str:
```

Comment:
Would we need to make the same change for phone number?

It still seems to be
```python
def _phone_number_raw_id(identifier):
    # type (PhoneNumberIdentifier) -> str
```

Reply:
(and then across all copies of the _shared folder)

Reply:
It was a private function so I cared less about types vs comments (existing stuff in the file uses comments).
Changed now.